js_modules/dagster-ui/packages/ui-core/src/assets/asset-checks/AssetChecks.tsx

```diff
@@ -86,24 +88,31 @@ export const AssetChecks = ({
         <Box flex={{direction: 'row', alignItems: 'center', gap: 6}}>
           <Body2>Latest materialization:</Body2>
 
-          <Tag icon="materialization">
-            {lastMaterializationDate ? (
-              <>
-                {formatDatetime(lastMaterializationDate, {
-                  month: 'short',
-                  day: 'numeric',
-                  year: isCurrentYear ? undefined : 'numeric',
-                })}{' '}
-                at{' '}
-                {formatDatetime(lastMaterializationDate, {
-                  hour: 'numeric',
-                  minute: 'numeric',
-                })}
-              </>
-            ) : (
-              'None'
-            )}
-          </Tag>
+          <Link
+            to={assetDetailsPathForKey(assetKey, {
+              time: lastMaterializationTimestamp,
+              view: 'events',
+            })}
+          >
+            <Tag icon="materialization">
+              {lastMaterializationDate ? (
+                <>
+                  {formatDatetime(lastMaterializationDate, {
+                    month: 'short',
+                    day: 'numeric',
+                    year: isCurrentYear ? undefined : 'numeric',
+                  })}{' '}
+                  at{' '}
+                  {formatDatetime(lastMaterializationDate, {
+                    hour: 'numeric',
+                    minute: 'numeric',
+                  })}
+                </>
```

Comment:
I'm a little skeptical of trying to concat these formatted pieces ourselves instead of using `TimestampDisplay` or something more consistent with the rest of the app.